src/main/java/org/jenkins/tools/test/model/hook/PluginCompatTesterHooks.java

```diff
@@ -103,19 +107,18 @@ private Map<String, Object> runHooks(String stage, Map<String, Object> elements)
         // Modifications build on each other, pertinent checks should be handled in the hook
         for(PluginCompatTesterHook hook : beforeHooks) {
             try {
-                System.out.println("Processing " + hook.getClass().getName());
                 if(!excludeHooks.contains(hook.getClass().getName()) && hook.check(elements)) {
+                    LOGGER.log(Level.INFO, "Running hook: {0}", hook.getClass().getName());
                     elements = hook.action(elements);
                     hook.validate(elements);
                 } else {
-                    System.out.println("Hook not triggered.  Continuing.");
+                    LOGGER.log(Level.FINE, "Skipping hook: {0}", hook.getClass().getName());
```

Comment:
We still print a message for skipped hooks but leave it to `FINE` level since this should typically only be relevant when debugging something while developing a hooks-related change.